shared/editor/embeds/Grist.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import Frame from "../components/Frame";
+import { EmbedProps as Props } from ".";
+
+const URL_REGEX = new RegExp(
+  "^https?://([a-z.-]+\\.)?getgrist\\.com/(.*)/p/([0-9]+)\\?embed=true$"
```

Comment:
Also, are you sure that all the pasted urls would always have `embed=true` at the end? If not, then you might want to loosen this a bit and later replace the trailing part of the url with `embed=true` as demonstrated [here](https://github.com/outline/outline/blob/74d9409cc32c5923fe72e98eb961d1d56304519e/shared/editor/embeds/GoogleDocs.tsx#L15)

Reply:
Yes, they must have embed=true at the end

Reply:
Not sure how Grist works but generally the idea with these embeds is that we take a regular url and transform it into the embedded version on the users behalf

Reply:
Ok, that would be useful actually, let me look at that